lib/rules/color-named/index.js

```diff
@@ -46,7 +48,14 @@ const rule = function(expectation, options) {
       return;
     }
 
-    const namedColors = Object.keys(namedColorData);
+    const namedColors = Object.keys(namedColorDataHex);
+    const namedColorData = {};
+    Object.entries(namedColorDataHex).forEach(([name, hex]) => {
```

Comment:
Travis [is failing](https://travis-ci.org/stylelint/stylelint/builds/275372897?utm_source=github_status&utm_medium=notification) on node@6 and node@4 as they [seemingly don't support](http://node.green/#ES2017-features-Object-static-methods-Object-entries). Can you please rewrite this snippet of code for node@4.2.1?

Reply:
Done!